components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -8758,9 +8757,10 @@ def getRenderingModels(self):
         """
 
         if not len(self._rm):
-            for m in [BlitzObjectWrapper(self._conn, m)
-                      for m in self._re.getAvailableModels()]:
-                self._rm[m.value.lower()] = m
+            query_service = self._conn.getQueryService()
+            for m in self._conn.getEnumerationEntries('RenderingModel'):
+                mod = query_service.get('RenderingModel', m.id)
```

Comment:
`getEnumerationEntries()` already makes a query to retrieve these models, why are we doing it again here and once for every model?

Reply:
OK, yes. Sorry - I had tried a bunch of things here (including using unloaded RenderingModel, which worked for rendering but not saving) and ended up getting mixed up between RenderingModels, Enumerations and various wrapper objects!
Sorted now...